scanners/opensecurity/mobsfscan/rules.yaml

```diff
@@ -371,6 +371,19 @@ rules:
       and retrieves the contents of this URL, but it does not sufficiently ensure
       that the request is being sent to the expected destination.
     ref: https://github.com/returntocorp/semgrep-rules/
+  CWE-919:
+    categories:
+    - ALL
+    - cwe-top-25
+    - cwe-919
```

Comment:
cwe-919 is not part of the existing list of categories. Here's the current list:
ALL
boost-baseline
boost-hardened
cloud-insecure-iam
cloud-resources-public-access
cloud-unencrypted-resources
cloud-weak-configuration
cloud-weak-secrets-management
cwe-1004
cwe-1022
cwe-1104
cwe-116
cwe-119
cwe-1236
cwe-125
cwe-1254
cwe-1275
cwe-1327
cwe-1333
cwe-1336
cwe-185
cwe-20
cwe-200
cwe-22
cwe-250
cwe-259
cwe-269
cwe-276
cwe-284
cwe-285
cwe-287
cwe-295
cwe-297
cwe-306
cwe-310
cwe-311
cwe-319
cwe-320
cwe-326
cwe-327
cwe-328
cwe-338
cwe-345
cwe-346
cwe-352
cwe-369
cwe-399
cwe-400
cwe-434
cwe-470
cwe-489
cwe-494
cwe-502
cwe-522
cwe-565
cwe-601
cwe-611
cwe-614
cwe-693
cwe-703
cwe-704
cwe-706
cwe-732
cwe-74
cwe-77
cwe-777
cwe-778
cwe-78
cwe-79
cwe-798
cwe-862
cwe-89
cwe-90
cwe-913
cwe-915
cwe-918
cwe-94
cwe-943
cwe-95
cwe-96
cwe-top-25
insufficient-logging
owasp-top-10
stored-secrets
supply-chain
supply-chain-cicd-severe-issues
supply-chain-cicd-vulnerable-pipeline
supply-chain-cicd-weak-configuration
supply-chain-missing-artifact-integrity-verification
supply-chain-scm-weak-configuration
vulnerable-and-outdated-components

For now I suggest we leave cwe-919 out of the list of categories and I will add it in the system, we can add it after here.